src/common/components/FiscalYearFilter/FiscalYearFilter.js

```diff
@@ -0,0 +1,32 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import { Loading } from '@folio/stripes/components';
+import { SelectionFilter } from '@folio/stripes-acq-components';
+import { usePayableFiscalYears } from '../../hooks';
+
+export const FiscalYearFilter = (props) => {
+  const { isLoading, fiscalYears } = usePayableFiscalYears();
```

Comment:
make sure package.json include required permission to fetch FY when user has permission to view invoices

Reply:
Do you mean this line? https://github.com/folio-org/ui-invoice/blob/master/package.json#L270

Reply:
it's permission to read FYs, but it's sub permission of `ui-invoice.payDifferentFY`, my request is to add permission in view invoices sub permissions

Reply:
actually to move from `payDifferentFY` to `view invoices` as pay without view is not possible

Reply:
Let me review it tomorrow. I have not worked with permissions yet

Reply:
@NikitaSedyx Is the permissions configured correctly?

Reply:
yes